nucleus/dataset.py

```diff
@@ -1862,3 +1851,75 @@ def query_items(self, query: str) -> Iterable[DatasetItem]:
         )
         for item_json in json_generator:
             yield DatasetItem.from_json(item_json)
+
+    def create_track(
```

Comment:
I think this should rather be on the `Scene` object abstraction. This is a bit too far away from where this would be used in my opinion.